src/pudl/validate.py

```diff
@@ -172,7 +174,12 @@ def check_max_rows(df, expected_rows=np.inf, margin=0.05, df_name=""):
     return df
 
 
-def check_min_rows(df, expected_rows=0, margin=0.05, df_name=""):
+def check_min_rows(
+    df: pd.DataFrame,
+    expected_rows: int | float = np.inf,
```

Comment:
The default value should probably be zero here, not infinity.

Reply:
Good catch, this seems like a mistake that never got changed back. Making it 0 now.